aries-site/src/examples/templates/page-layouts/StickyHeaderExample.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Header, Box, Footer, Main, ResponsiveContext, Text } from 'grommet';
+
+export const StickyHeaderExample = () => {
+  return (
+    <AppContainer overflow="auto">
+      <Box fill>
+        {/* <Box height={{ min: '100%' }} gap="small"> */}
```

Comment:
1) Wondering if we should leave the gap in here? The lack thereof makes me think there is an error
<img width="1169" alt="Screen Shot 2020-06-22 at 1 20 59 PM" src="https://user-images.githubusercontent.com/1756948/85326893-40de8480-b48b-11ea-9efd-e9e2286398ae.png">

2) Remove commented code

Reply:
The reason I had left it out is that if we think people will copy and paste the code, having a gap there would create a strange scrolling experience where content gets hidden before it touches the header. Agree though that upon first glance it feels out of sync with the rest of examples.

Reply:
Sounds good. We can reevaluate if we get that feedback from anywhere else.